File: src/main/java/info/szadkowski/contact/ContactServiceApplication.java
package info.szadkowski.contact;

import info.szadkowski.contact.configuration.MainConfiguration;
import info.szadkowski.contact.controller.MessageController;
import info.szadkowski.contact.properties.MailAddressesProperties;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackageClasses = {
        MainConfiguration.class,
        MessageController.class,
        MailAddressesProperties.class
})
public class ContactServiceApplication {
  public static void main(String[] args) {
    SpringApplication.run(ContactServiceApplication.class, args);
  }
}

File: src/main/kotlin/info/szadkowski/contact/ContactServiceApplication.kt
package info.szadkowski.contact

import info.szadkowski.contact.configuration.MainConfiguration
import info.szadkowski.contact.controller.MessageController
import info.szadkowski.contact.properties.MailAddressesProperties
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication(
    scanBasePackageClasses = [
        MainConfiguration::class,
        MessageController::class
    ]
)
@ConfigurationPropertiesScan(
    basePackageClasses = [MailAddressesProperties::class]
)
class ContactServiceApplication

fun main(args: Array<String>) {
    runApplication<ContactServiceApplication>(*args)
}
